packages/publish/src/publish.ts

```diff
@@ -197,9 +355,14 @@ handler.publish = async (npmRc: string, pkgPath: string, app: Application) => {
       app.log.warn(err.message);
     }
     app.log.info(`publishing ${pkgPath}`);
+    // If this is a prerelease, we publish to the "next" dist tag on npm:
+    const publishOpts = ['publish', '--access=public'];
+    if (prerelease) {
+      publishOpts.push('--tag=next');
```

Comment:
maybe move `next` to a variable? It occurs multiple times in the code.

Reply:
if we decide to expand it later, we might as well want to allow tags like `publish:legacy-10` (then `publish:candidate` would logically become `publish:next`)

Reply:
...actually, extracting tags from the label should not be too hard to do right now?

Reply:
talked offline, let's make this follow up work (supporting multiple labels).